CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+Contribution Guidelines
+=======================
+
+First of all, each single contribution is appreciated, whether a typo fix,
+improved documentation, a fixed bug or a whole new feature.
+
+## Feature request
+
+If you think a feature is missing, please report it or implement it. If you report it, describe the more
+precisely what you would like to see implemented. It would be nice if you can do
+some search before submitting it and link the resources to your description.
+
+## Bug report
+
+If you think you have detected a bug or a doc issue, please report it (open [issue][issue]) or even better fix it. If you report it,
+please be the more precise possible. Here a little list of required information:
+
+ * Symfony-standard fork which reproduces the bug.
+ * Precise description of the bug.
+ * Symfony version used.
+ * Bundle version used.
+
+ ## Bug fix
+
+Fork the repository, clone it and create a new branch with the following commands:
+
+ ``` bash
+ $ git clone git@github.com:liip/LiipImagineBundle.git
+ $ git checkout -b bug-fix-description
+ ```
+
+ Then, install the dependencies through [Composer][composer]
+
+ ``` bash
+ $ composer install
+ ```
+
+ When you're on the new branch with the dependencies, code as much as you want and when the fix is ready,
+ you will need to add tests and update the documentation. Everything is tested with
+ [PHPUnit][php-unit], documentation formatted with markdown under the `Resources/doc` directory.
```

Comment:
`formatted with markdown` -> `formatted with reStructuredText`  (and maybe you can link to http://symfony.com/doc/current/contributing/documentation/format.html#restructuredtext)